### ISSUE TITLE
SK seems to be listed for all countries even when not installed 

### ISSUE THREAD

**Issue by jonasbn** [jonasbn/perl-date-holidays#45]
# Issue Template

### Description

When calling `is_holiday` for all available countries, `sk` is in the resultset even if not installed.

# Bugs and Issues

### Specify what version was used

Version used 1.25

### Supply some information on the environment

This is perl 5, version 30, subversion 0 (v5.30.0) built for darwin-2level
(with 1 registered patch, see perl -V for more detail)

### Expected behaviour

I did not expect `sk` to be listed, when I request `da` and `no``

### Actual and observed behaviour

I get the following structure back (encoded as JSON):

```json
{"dk":"Nytårsdag","no":"nyttårsdag","sk":""}
```

### Steps to reproduce the behaviour

```perl
use strict;
use warnings;
use Date::Holidays;
use Locale::Country; # all_country_codes
 
my @country_codes = all_country_codes();

my $dh = Date::Holidays->new( countrycode => 'dk' );

my $holidays_hashref = Date::Holidays->is_holiday(
        year    => 2020,
        month => 1,
        day      => 1,
        countries => \@country_codes,
);

if (exists $holidays_hashref->{'sk'}) {
    print 'Checking for presence of SK';
}
```

**Comment by jonasbn** [comment]
The currently implemented test, will break if Date::Holidays::SK is installed. One test assumes it presence, where another assumes it's lacking presence. Manipulating it's load status in `%INC` does not suffice.

I am working in a conditional overload of the `_load` method in Date::Holidays::Adapter emulating that Date::Holidays::SK is not installed and hence both scenarios can be tested

**Comment by jonasbn** [comment]
Included in release 1.27
